src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataObject.cs

```diff
@@ -13,15 +13,15 @@ public sealed class ExtensionDataObject
     {
         private IList<ExtensionDataMember> _members;
 
-#if USE_REFEMIT
+#if USE_REFEMIT || NET_NATIVE
```

Comment:
I'd prefer keep them as is for now as we have other directives like this in other files. We can clean them all later.